client/src/components/SignUp.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import { Button } from 'react-bootstrap';
+import { useNavigate } from 'react-router-dom';
+import { postData } from '../api';
+
+export default function Login() {
```

Comment:
guessing you just missed this but i'd rename this component to SignUp

Reply:
yeah lol